lib/bamboo/adapters/sendgrid_adapter.ex

```diff
@@ -0,0 +1,160 @@
+defmodule Bamboo.SendgridAdapter do
+  @moduledoc """
+  Sends email using SendGrid's JSON API.
+
+  Use this adapter to send emails through SendGrid's API. Requires that an API
+  key is set in the config.
+
+  ## Example config
+
+      # In config/config.exs, or config.prod.exs, etc.
+      config :my_app, MyApp.Mailer,
+        adapter: Bamboo.SendgridAdapter,
+        api_key: "my_api_key"
+
+      # Define a Mailer. Maybe in lib/my_app/mailer.ex
+      defmodule MyApp.Mailer do
+        use Bamboo.Mailer, otp_app: :my_app
+      end
```

Comment:
Perhaps. It seems a little odd to put that in the docs. How's the bottom of the README?